public/static/js/spawned_hosts.js

```diff
@@ -92,6 +70,68 @@ mciModule.controller('SpawnedHostsCtrl', ['$scope','$window', '$timeout', 'mciSp
     }
 
 
+
+    $scope.updateSpawnedHosts = function() {
+        mciSpawnRestService.getSpawnedHosts(
+            'hosts', {}, {
+                success: function(resp) {
+                    var hosts = resp.data;
+                    _.each(hosts, function(host) {
+                        for(var i = 0; i < $scope.hosts.length; i++) {
+                            if ($scope.hosts[i].id !== host.id && $scope.hosts[i].id !== host.tag) {
+                                continue;
+                            }
+                            $scope.computeUptime(host);
+                            $scope.computeExpirationTimes(host);
+                            $scope.hosts[i].uptime = host.uptime;
+                            $scope.hosts[i].expires_in = host.expires_in;
+                            $scope.hosts[i].status = host.status;
+                            $scope.hosts[i].id = host.id;
+                            if ($scope.hosts[i].instance_type === undefined || $scope.hosts[i].instance_type === "") {
+                                $scope.hosts[i].instance_type = host.instance_type;
+                            }
+                            if ($scope.hosts[i].selectedInstanceType === undefined || $scope.hosts[i].selectedInstanceType === "") {
```

Comment:
I don't think we should have this field refresh since it's an editable control. Even though we're checking if it's populated, I think it'd be jarring

Reply:
I know what you're saying, but this information isn't available until the instance is Running, and once the instance is running if the user wants to then change the type they need to see the existing type.

Reply:
(Although they'd have to Stop the instance first but still).